flink-state-backends/flink-statebackend-changelog/src/test/java/org/apache/flink/state/changelog/ChangelogPqStateTest.java

```diff
@@ -40,57 +40,55 @@
 
 import static java.util.Collections.emptyList;
 import static java.util.Collections.singletonList;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** ChangelogKeyGroupedPriorityQueue Test. */
 @SuppressWarnings({"rawtypes", "unchecked"})
 public class ChangelogPqStateTest {
```

Comment:
```suggestion
class ChangelogPqStateTest {
```
could be package private